libsolidity/ast/ASTJsonConverter.cpp

```diff
@@ -563,7 +564,8 @@ bool ASTJsonConverter::visit(InlineAssembly const& _node)
 
 	Json::Value externalReferencesJson = Json::arrayValue;
 
-	for (auto&& it: boost::range::sort(externalReferences))
+	ranges::sort(externalReferences);
```

Comment:
Does this change it in place? It looked like the previous one was pure

Reply:
yes, both of them sort the container in place. look at this [example](http://coliru.stacked-crooked.com/a/18c07a3c138d6218)

Reply:
boost version returns a reference of the container. it is simpler to use but confusing; at first glance it looks like it is pure an sort lazy, but we don't have lazy sort in boost or std or range-v3.